airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/commands.py

```diff
@@ -3,14 +3,34 @@
 #
 
 import asyncclick as click
+import semver
 from pipelines.airbyte_ci.connectors.bump_version.pipeline import run_connector_version_bump_pipeline
 from pipelines.airbyte_ci.connectors.context import ConnectorContext
 from pipelines.airbyte_ci.connectors.pipeline import run_connectors_pipelines
 from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
 
 
+class BumpType(click.ParamType):
+    name = "bump-type"
+
+    def __init__(self):
+        self.choices = ["patch", "minor", "major"]
+
+    def convert(self, value, param, ctx):
+        if value in self.choices:
+            return value
+        if value.startswith("version:"):
+            version_str = value.split("version:", 1)[1]
+            if semver.VersionInfo.is_valid(version_str):
```

Comment:
Do we care that this allows for e.g. dev versions, alpha versions etc.? My guess is probably not, since we don't intend it to be used much.